models/Import/Service/ResultImporter.php

```diff
@@ -232,6 +238,10 @@ private function getTestScoreVariables(AbstractRdsResultStorage $resultStorage,
             }
         }
 
+        if (!isset($scoreTotalVariable)) {
+            return null;
+        }
+
         if (!isset($scoreTotal, $scoreTotalVariableId, $scoreTotalMax, $scoreTotalVariable)) {
```

Comment:
You can remove $scoreTotalVariable from this if because you checked it previously